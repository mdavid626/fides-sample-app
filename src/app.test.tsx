import React from 'react';
import { render } from '@testing-library/react';
import App from './app';

describe('app', () => {
  it('should render', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
