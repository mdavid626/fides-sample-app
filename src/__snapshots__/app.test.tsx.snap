// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should render 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    Test
  </div>
</DocumentFragment>
`;
